libs/sdk-ui-tests/scenarios/pivotTable/totals.tsx

```diff
@@ -89,4 +89,8 @@ export default scenariosFor<IPivotTableProps>("PivotTable", PivotTable)
     })
     .addScenario("two measures and grand totals and multiple subtotals", {
         ...PivotTableWithTwoMeasuresGrandTotalsAndSubtotals,
+    })
+    .addScenario("two measures and grand totals and multiple subtotals sorted by second attribute", {
```

Comment:
This will add a new screenshot, you need to commit it. To get it, run the `extended test` and check the report for the new image (sadly M1 Macs cannot run this locally yet).

Reply:
Also it needs new snapshots, this is why the build is failing. Run `npm run test-once` in the sdk-ui-tests folder. This should create the snapshots.